src/Swarm/Game/Step.hs

```diff
@@ -417,8 +418,8 @@ stepCESK cesk = case cesk of
   In (TAntiInt v) _ s k ->
     return $ Up (Fatal (T.append "Antiquoted variable found at runtime: $int:" v)) s k
   -- Require and requireDevice just turn into no-ops.
-  In (TRequireDevice {}) e s k -> return $ In (TConst Noop) e s k
-  In (TRequire {}) e s k -> return $ In (TConst Noop) e s k
+  In TRequireDevice {} e s k -> return $ In (TConst Noop) e s k
+  In TRequire {} e s k -> return $ In (TConst Noop) e s k
```

Comment:
Did fourmolu insist on getting rid of these parens?  I understand intellectually why they are not needed, but I kinda hate it.  `In TRequire {} e s k` really looks like `In` is taking 5 parameters.

Reply:
My HLint does not like it and I got tired of it mentioning it. I will change it to `(TRequire _num _ent)`.

Reply:
We can also turn off specific hlint hints in `hlint.yaml`.

Reply:
In general, I think it is useful to remove extra parenthesis, but this is an edge case.

Presumably, it is different between some HLint versions because CI did not warn about it.